src/unity/python/turicreate/test/test_sarray.py

```diff
@@ -9,6 +9,7 @@
 from __future__ import division as _
 from __future__ import absolute_import as _
 from ..data_structures.sarray import SArray
+from .. import load_sarray
```

Comment:
This should be imported directly from SArray, with `from ..data_structures.sarray import load_sarray`.